.github/workflows/shaptools-ci.yml

```diff
@@ -0,0 +1,103 @@
+name: Formula CI
+# - this workflow will
+# - test python version matrix [2.7,3.6, 3.7, 3.8, 3.9] for coverage 
+# - uploads coverage data to codeClimate --> Results are linked  as a  badges in the README
+# - deliver the package content to the configured repository
+# - submit the new package content to the upstream repository
+on: [push, pull_request]
+env:
+  PACKAGE_NAME: python-shaptools
+  TAR_NAME: shaptools
+jobs:
+  
+  testPython:
+    runs-on: ubuntu-18.04
+    if: ${{ github.event_name != 'pull_request' }} 
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [2.7,3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}    
+    - name: Test python version [2.7,3.6, 3.7, 3.8, 3.9] and install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
```

Comment:
Change this to `pip install pytest pytest-cov mock`.
Pylint is not used, so we can remove it

Reply:
Yes, true that.